src/vm/hints/hint_utils.rs

```diff
@@ -739,6 +739,76 @@ pub fn is_positive(
     }
 }
 
+//Implements hint:
+// %{
+//     from starkware.cairo.common.math_utils import assert_integer
+//     assert ids.MAX_HIGH < 2**128 and ids.MAX_LOW < 2**128
+//     assert PRIME - 1 == ids.MAX_HIGH * 2**128 + ids.MAX_LOW
```

Comment:
These two asserts are for constants that are defined before the hint execution
https://github.com/starkware-libs/cairo-lang/blob/167b28bcd940fd25ea3816204fa882a0b0a49603/src/starkware/cairo/common/math.cairo#L122
These constants are in the "identifiers" field of the .json file generated from the complied Cairo program.
Since we are not deserializing the "identifiers" field of the .json file yet,  I assign them directly in the hint implementation and avoid the two asserts

```
//     assert ids.MAX_HIGH < 2**128 and ids.MAX_LOW < 2**128
//     assert PRIME - 1 == ids.MAX_HIGH * 2**128 + ids.MAX_LOW
```